src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -343,12 +337,23 @@ object ScalafixPlugin extends AutoPlugin {
 
   private def scalafixInputTask(
       config: Configuration
-  ): Def.Initialize[InputTask[Unit]] =
+  ): Def.Initialize[InputTask[Unit]] = {
+    val parser = Def.setting(
+      new ScalafixCompletions(
+        workingDirectory = (ThisBuild / baseDirectory).value.toPath,
+        loadedRules = () => scalafixInterfaceProvider.value().availableRules(),
+        terminalWidth = Some(JLineAccess.terminalWidth),
+        allowedTargetFilesPrefixes =
+          (scalafix / unmanagedSourceDirectories).value.map(_.toPath)
```

Comment:
we don't rely on `unmanagedSources` here as
1. we want to be able to target directories as well
2. it's a task, not available at settings evaluation time anyway